_posts/2022-04-20-unmap-me-project.markdown

```diff
@@ -61,6 +61,48 @@ Admin:
 
 ## Design Specifications
 
+### Data Model
```

Comment:
We need another table for unmap memberships - to track admins/owners of an unmap, and if its private, members.

Stretch goal: an audit table to see who created/edited/deleted what locations when.

Reply:
I didn't want to dive into memberships until permissions was discussed.

An audit log is something we might be able to accomplish without it being a whole table... depending on what we're trying to get out of it.